docs/self-managed/components/orchestration-cluster/core-settings/concepts/schema-and-migration.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
Can you clarify which sections of this can be removed, or direct me toward someone who can confidently confirm what may be removed here?